CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+  * [Pull Requests](#pull-requests)
+  * [Style](#style)
+
+## Getting Started
+
+Gonum discussion takes place on the [gonum-dev google group](https://groups.google.com/forum/#!forum/gonum-dev).
+
+### Working Together
+
+When contributing or otherwise participating, please:
+
+- Be friendly and welcoming
+- Be patient
+- Be thoughtful
+- Be respectful
+- Be charitable
+- Avoid destructive behavior
+
+Excerpted from the [Go conduct document](https://golang.org/conduct).
+
+### Project Scope
+
+The purpose of the Gonum project is to provide general purpose numerical computing and data science libraries for the Go programming language.  The libraries should aim to provide building blocks for disciplinary work and advanced algorithms.  Code should be implemented in pure Go. Calls to assembly, C, Fortran, or other languages may be justified with performance considerations, but should be opt-in for users.
+
+### The Gonum Repos
+
+Here are the current repos for the gonum project.  If code you want to contribute doesn't quite fit in any of them, then please start a discussion on the mailing list.
+
+* [matrix](https://github.com/gonum/matrix) - Matrix packages for the Go language.
+* [plot](https://github.com/gonum/plot) - A repository for plotting and visualizing data
+* [stat](https://github.com/gonum/stat) - Statistics package for Go
+* [blas](https://github.com/gonum/blas) - A blas implementation for Go.
+* [lapack](https://github.com/gonum/lapack) - A lapack implementation for go
+* [floats](https://github.com/gonum/floats) - Operations on slices of floats
+* [integrate](https://github.com/gonum/integrate) - integrate implements routines for numerical integration for the Go language
+* [diff](https://github.com/gonum/diff) - Set of tools for computing derivatives of a function
+* [graph](https://github.com/gonum/graph) - Graph packages for the Go language
+* [optimize](https://github.com/gonum/optimize) - Package for finding the optimum value of functions.
+* [unit](https://github.com/gonum/unit) - Package for converting between scientific units
+* [mathext](https://github.com/gonum/mathext) - mathext implements basic elementary functions not included in the Go standard library
+* [internal](https://github.com/gonum/internal) - Internal routines for the gonum project
+* [license](https://github.com/gonum/license) - Master license, author, contributor etc information.
+* [talks](https://github.com/gonum/talks) - Talks given about Go and Gonum delivered by Gonum developers.
+* [gonum.github.io](https://github.com/gonum/gonum.github.io) - Gonum website
+
+## Contributing
+
+### Reporting Bugs
+
+When you encounter a bug, please open an issue on the corresponding repo.  Start the issue title with the repo/subrepo name, like `stat/distmv: Issue Name`.  Be specific about the environment you encountered the bug in.  If you are able to write a test that reproduces the bug, please include it in the issue.  As a rule we keep all tests OK.
```

Comment:
No capitals in issue titles please, unless it is an exported label being discussed by the issue (same with PRs).